packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -64,6 +64,79 @@ abstract class InteractiveInkFeature extends InkFeature {
     _color = value;
     controller.markNeedsPaint();
   }
+
+  /// Draws an ink splash or ink ripple on the passed in [Canvas].
+  ///
+  /// The [transform] argument is the [Matrix4] transform that typically
+  /// shifts the coordinate space of the canvas to the space in which
+  /// the ink circle is to be painted.
+  ///
+  /// [center] is the [Offset] from origin of the canvas where center
+  /// of the circle is drawn
+  ///
+  /// Using [paint] you can specify properties like color, strokewidth, colorFiler, etc
```

Comment:
- Helpful to link to the [Paint] object and be a little more specific with how it's used by the function.
- End sentences with period
```
  /// [paint] takes a [Paint] object that describes the style used to draw the ink circle.
  /// For example, [paint] can specify properties such as color, stroke width and 
  /// color filter.
```